src/App.tsx

```diff
@@ -1,7 +1,35 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  message: string,
+};
+
+export class App extends React.Component<{}, State> {
+  state: State = {
+    message: '',
+  };
+
+  componentDidMount(): void {
+    this.setState({ message: 'Nothing was pressed yet' });
+  }
+
+  componentDidUpdate(): void {
+    document.addEventListener('keyup', this.handleKeyUp);
+  }
+
+  componentWillUnmount(): void {
+    document.removeEventListener('keyup', this.handleKeyUp);
+  }
+
+  handleKeyUp = (event: KeyboardEvent) => {
+    this.setState({ message: `The last pressed key is [${event.key}]` });
+  };
+
+  render() {
+    return (
+      <div className="App">
+        <p className="App__message">{this.state.message}</p>
```

Comment:
Just put 'Nothing was pressed yet' here within the ternary. Hook the whole thing to pressedKey.